src/AssetsPage/CopyButton/CopyButton.test.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import CopyButton from './index';
+
+const defaultProps = {
+  label: 'I am a copy button!',
+  onCopyLabel: 'Copied!',
+  textToCopy: 'This is my copy text!',
+};
+
+describe('<CopyButton />', () => {
+  let wrapper;
+  let button;
+
+  beforeEach(() => {
+    jest.mock('copy-to-clipboard');
```

Comment:
I wasn't able to find a way to both mock the module and be able to see how often the function is being called. It would be nice to verify that the CopyButton calls the copy function once per click, but I need to look into how to do that with a manual mock, since I'm only familiar with doing that with mocked functions.